listlayout.py

```diff
@@ -1,24 +1,53 @@
 from layout import Layout
 from pos import Pos
+from size import Size
 
 import pygame
 import colour
 
 class ListLayout(Layout):
-    def __init__(self, size, pos=Pos(0,0)):
-        super(ListLayout, self).__init__(size, pos)
+    def __init__(self, size, pos=Pos(0,0), border=5, orientation="vertical", spacing=2):
+        super(ListLayout, self).__init__(size, pos, border)
         self.next_child_origin = Pos(
-            self.origin.x + self.pos.x, 
-            self.origin.y + self.pos.y
+            self.origin.x + self.pos.x + self.border, 
+            self.origin.y + self.pos.y + self.border
         )
+        self.orientation = orientation
+        self.spacing = spacing
 
     def add(self, child):
+
+        if not self.children:
+            self.size = Size(
```

Comment:
when adding a border should we add additional width and height to the advertised value or should we keep the width and height but reduce the internal width and height?